packages/react-component-library/src/components/Alert/Alert.tsx

```diff
@@ -60,19 +61,40 @@ export const Alert: React.FC<AlertProps> = ({
     `rn-alert__description--${variant}`
   )
 
+  const titleId = getId('alert-title')
+  const descriptionId = getId('alert-description')
+
   return (
     !closed && (
-      <div className={classes} data-testid="alert">
-        <div className={iconClasses} data-testid="state-icon">
+      <div
+        aria-describedby={descriptionId}
+        aria-labelledby={titleId}
+        className={classes}
+        data-testid="alert"
+        role="alert"
+      >
+        <div
+          aria-hidden="true"
+          className={iconClasses}
+          data-testid="state-icon"
+        >
           {VARIANT_ICON_MAP[variant]}
         </div>
         <div className="rn-alert__content" data-testid="content">
           {title && (
-            <h2 className="rn-alert__title" data-testid="content-title">
+            <h2
+              className="rn-alert__title"
+              data-testid="content-title"
+              id={titleId}
```

Comment:
Can you explain this to me quickly - is the unique ID added to help screen readers?

Reply:
Yes, the screenreader knows that the `div` is an `alert` and then looks for the associated label and description.